deepchem/feat/tests/test_mxmnet_featurizer.py

```diff
@@ -0,0 +1,104 @@
+"""
+Test for MXMNet Featurizer class.
+"""
+
+import deepchem as dc
+import numpy as np
+import unittest
+
+edge_index = {
+    "C1=CC=NC=C1":
+        np.asarray([[0, 1, 1, 5, 5, 2, 2, 4, 4, 3, 3, 0],
+                    [1, 0, 5, 1, 2, 5, 4, 2, 3, 4, 0, 3]]),
+    "CC(=O)C":
+        np.asarray([[0, 3, 3, 2, 3, 1], [3, 0, 2, 3, 1, 3]]),
+    "C":
+        np.empty((2, 0), dtype=int)
+}
+
+node_features = {
+    "C1=CC=NC=C1": np.asarray([[1.], [1.], [2.], [1.], [1.], [1.]]),
+    "CC(=O)C": np.asarray([[1.], [1.], [3.], [1.]]),
+    "C": np.asarray([[1.]])
+}
+
+
+class Test_MXMNet_Featurizer(unittest.TestCase):
```

Comment:
Make sure to use camel case `TestMXMNetFeaturizer`

Reply:
done